lib/std/log.zig

```diff
@@ -110,13 +111,46 @@ pub const default_level: Level = switch (builtin.mode) {
     .ReleaseSmall => .err,
 };
 
+const config = if (@hasDecl(root, "log_config"))
+    root.log_config
+else
+    default_config;
+
```

Comment:
Should this config approach be used? Note: this isn't implemented right now, I just left this in as a stub.